### PR TITLE
Updated dependency

### PR DESCRIPTION
**Pull request by sudhirverma**

### CODE REVIEW

File: test/odo.int.ts [hunk 1]
```diff
@@ -12,10 +12,12 @@ import { Cluster } from '../src/openshift/cluster';
 suite('odo integration', () => {
```
Comment (dgolovin) [inline test/odo.int.ts]: Only  changes in package.json is good to go. Everything else does not add any value .

File: test/odo.int.ts [hunk 2]
```diff
@@ -12,10 +12,12 @@ import { Cluster } from '../src/openshift/cluster';
...
     const oi: odo.Odo = odo.getInstance();
     let sb: sinon.SinonSandbox;
+    let showInformationMessageStub: sinon.SinonStub;
 
     setup(() => {
         sb = sinon.createSandbox();
-        sb.stub(vscode.window, 'showInformationMessage').resolves('Download and install');
```
Comment (dgolovin) [inline test/odo.int.ts]: Why replace single line with two with exactly the same result?

File: test/openshift/application.test.ts [hunk 3]
```diff
@@ -22,8 +22,8 @@ suite('Openshift/Application', () => {
     let sandbox: sinon.SinonSandbox;
     let execStub: sinon.SinonStub;
     let getProjectNamesStub: sinon.SinonStub;
-    const projectItem = new TestItem(null, 'project');
-    const appItem = new TestItem(projectItem, 'app');
+    const projectItem: OpenShiftObject = new TestItem(null, 'project');
```
Comment (dgolovin) [inline test/openshift/application.test.ts]: No need to declare type for direct assignment, it is going to be inferred during the compilation.

Comment (dgolovin) [reply test/openshift/application.test.ts]: All the changes below have the same problems.

File: test/odo.int.ts (hunk 2 above)
Comment (sudhirverma) [reply test/odo.int.ts]: after updating package.json it was giving an error for tslint that why I have replaced the single line.

Comment (dgolovin) [reply test/odo.int.ts]: could you fix errors by configuring rules for linter?

**Pull request merged**